packages/fxa-auth-server/lib/routes/subscriptions/stripe.ts

```diff
@@ -87,6 +88,7 @@ export class StripeHandler {
   ) {
     this.subscriptionAccountReminders =
       require('../../subscription-account-reminders')(log, config);
+    this.automaticTax = !!config.subscriptions.stripeAutomaticTax?.enabled;
```

Comment:
Looks like you can remove the optional chain operator here, since our default value is defined as `false`? Just checked locally.